docs/architecture/decisions/0025-caching

```diff
@@ -0,0 +1,71 @@
+# 24. Production Release Cadence
+
+Date: 2024-14-02
+
+## Status
+
+In Review
+
+## Context
+
+We experienced problems with our Cloudfront caching infrastructure early in our November launch.  In response, we turned off caching across the application.  We would like to utilize caching again without incurring the same issues.
+
+Details:
+Originally, Cloudfront was utilized to provide caching capabilities in our application.  All incoming HTTP requests first go through a Cloudfront endpoint, which has a caching infrastructure enabled by default. Cloudfront then decides whether to pass each request to our running Django app inside cloud.gov or if it will respond to with cached data.  The big problem with this feature is Cloudfront's caching has a default timeout of 24-hours, which we cannot control.  This led to issues on our November launch;  Incidents reported include the following...
+ - Users couldn't utilize login.gov properly and had to wait a day before they would be able to login.  This was traced back to the 24-hour cache timeout.
+ - Changes made by admins would not be reflected in the app (due to the cached data not updating)
+
+To resolve these issues, we added "no cache" headers throughout our application.  Currently, every single HTTP response that comes from Django says "Cache control: no cache" in the headers, which instructs Cloudfront not to cache the associated data.  This effectively removes Cloudfront caching for us.
```

Comment:
**Comment:** Would it be worth adding a code snippet of what "no cache" headers look like for reference?